src/battle-log.ts

```diff
@@ -742,6 +744,10 @@ class BattleLog {
 				}
 			}
 			let dataUri = '';
+			attribs = html.sanitizeAttribs(tagName, attribs, (urlData: any) => {
+				if (urlData.scheme_ === 'geo' || urlData.scheme_ === 'sms' || urlData.scheme_ === 'tel') return null;
+				return urlData;
+			});
```

Comment:
Moving this here ruins all the other behavior of this function (the `img` data-URI sanitization no longer works, for instance)